### ISSUE TITLE
a11y view HTML template should have the iframe scroll with you as you look at the pDOM

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#617]

**Comment by zepumph** [comment]
@jessegreenberg could you please review by using an a11y view from master in a few browsers and seeing how you like the new feature?

**Comment by jessegreenberg** [comment]
@zepumph great idea, this is really cool, thank you! This is going to be really nice in development testing and presentations.

Working great in FF and Chrome,  but IE, Edge and Safari don't support classList.replace(). Wondering if we can implement that ourselves?

**Comment by zepumph** [comment]
Arg good catch! Anything else?

**Comment by jessegreenberg** [comment]
@zepumph its working great in IE11, Edge, Chrome, Firefox, and Safari! I have been using this a bunch already this week, it has been very nice. Closing.

**Comment by zepumph** [comment]
Note that this solution doesn't work well on IE, it get's stuck in the scroll view when you scroll back to the top. Leaving open and on hold depending on how https://github.com/phetsims/a11y-research/issues/63 goes.

**Comment by zepumph** [comment]
in phetsims/a11y-research#66 I ditched this method and now have the left column fixed (like phet-io studio). It works well, and I hope we don't have to go back to this approach. Closing
